tests/trainer/test_fsdp.py

```diff
@@ -31,7 +36,14 @@ def test_fsdp_device_initialization(model: ComposerClassifier, mixed_precision:
 
     """
     num_classes = 10
-    model = model(num_features=num_classes, device=device)
+
+    resolved_device = device
+    if device == 'mixed':
+        if dist.get_local_rank() == 0:
+            resolved_device = 'cpu'
+        else:
+            resolved_device = 'meta'
+    model = model(num_features=num_classes, device=resolved_device)
```

Comment:
comment
this + `sync_module_states` is how meta init works.
TIL 😄

Reply:
*mixed, but yeah haha